### ISSUE TITLE
Possible conflict with newer builds of the twilight forest

### ISSUE THREAD

**Issue by Sunconure11** [DimensionalDevelopment/JustEnoughIDs#52]
https://github.com/TeamTwilight/twilightforest/issues/685

**Comment by Xalcon** [comment]
This is caused by a change to [BlockTFMagicLogSpecial#sendChangedBiome()](https://github.com/TeamTwilight/twilightforest/blob/1.12.x/src/main/java/twilightforest/block/BlockTFMagicLogSpecial.java#L158)
The method now accepts an additional Biome parameter and because of this, the mixin library crashes with a InvalidMixingException. This sounds like an easy fix but I'm not sure if there is more that needs to be done. The sendChangedBiome method also changed slighly since it now passes `pos.getY()` instead of `128.0` around.
https://github.com/TeamTwilight/twilightforest/commit/a8c4575fe3e73bce257cec35d505e2b7a458f4db#diff-3726bc9a77a4b68c44c0c3bdeb7cd63eR159

**Comment by TacticalUniverse** [comment]
@Runemoro Any updated on if this or other JustEnoughIDs issues are being fixed or worked on

**Comment by Modpack-Support** [comment]
@ZombieHDGaming any update is this is being worked on

**Comment by gudenau** [comment]
I also have this issue, what is the latest version of The Twilight Forest that supports this?

**Comment by viceroy369** [comment]
Under what circumstance does this crash occur? I tried the newest TF+JEID in single player and it seems to be working just fine??

**Comment by Xalcon** [comment]
@viceroy369 are you sure you used the latest versions of both? Twilight Forest 3.8.689 does indeed work, but it's not the **latest** version.
Installing the minimal setup with the following versions will crash on startup:
* JustEnoughIDs-1.0.2-26
* twilightforest-1.12.2-3.9.851-universal
  * CTM-MC1.12.2-0.3.3.22
  * Patchouli-1.0-17
It even crashes if you remove twilight forests dependencies (I assume the crash happens before forge could check the dependencies)
```
[12:48:03] [Client thread/FATAL]: Mixin apply failed mixins.jeid.modsupport.json:MixinBlockTFMagicLogSpecial -> twilightforest.block.BlockTFMagicLogSpecial: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException Overwrite target "sendChangedBiome" was not located in target class twilightforest/block/BlockTFMagicLogSpecial
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: Overwrite target "sendChangedBiome" was not located in target class twilightforest/block/BlockTFMagicLogSpecial
```

**Comment by viceroy369** [comment]
Derp. Hadn't notice TF had a new build out. Yes the latest version crashes.

**Comment by johnneijzen** [comment]
same here if use latest version TF is crashes but release version works fine

**Comment by TheRealDadbeard** [comment]
any update on this one?

**Comment by Sunconure11** [comment]
There's an add on for JEID that fixes this

**Comment by TheRealDadbeard** [comment]
huh okay where?? Is there any addon too for fixing broken biomes names? It was a long time since this has been updated and I can't just remove it...

**Comment by ZombieHDGaming** [comment]
Fixed in latest commits.

**Comment by Saereth** [comment]
any chance on a new cf build for this soon? last cf build was like october of last year.

**Comment by DeadSix27** [comment]
You could just use the GH release:
<https://github.com/DimensionalDevelopment/JustEnoughIDs/releases>

BUT those seem to not work for me. I still have to use JEID (old ver) + that JEID addon to fix this.

**Comment by Saereth** [comment]
whats the jeid addon that uses it? and no cant release a pack on twitch with unsupported builds (on cf), well you can but you gotta go through the 3rd party permission system which is a pita

**Comment by DeadSix27** [comment]
there: https://minecraft.curseforge.com/projects/justenoughids-integration-jeidsi/files

**Comment by ZombieHDGaming** [comment]
@DeadSix27 are you sure you're even using the latest release for the JEID dev builds? Cause JEIDsI won't do anything since those fixes were combined with ours with the permission of the dev who wrote the addon

**Comment by DeadSix27** [comment]
@ZombieHDGaming Yes, I've even build it myself from git master to make sure, doesn't work still.

**Comment by ZombieHDGaming** [comment]
Once the dev build is done building, give that a try. It should work perfectly now from my tests.
